### PR TITLE
Moved AWS_API_CALL_TerminateInstances from addition to deletion category

### PR DESCRIPTION
**Pull request by juliancheal**
This fixes BZ https://bugzilla.redhat.com/show_bug.cgi?id=1537165

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -8,8 +8,10 @@
           - AWS_EC2_Instance_CREATE
           - AWS_EC2_Instance_UPDATE
           - AWS_EC2_Instance_DELETE
```
Comment (bronaghs) [inline config/settings.yml]: @juliancheal - Can you move AWS_EC2_Instance_DELETE while you are at it?

**Comment by juliancheal** [comment]
@bronaghs Travis all green

**Comment by juliancheal** [comment]
@miq-bot add gaprindashvili/yes

**Comment by juliancheal** [comment]
@miq-bot add_label blocker

**Comment by juliancheal** [comment]
@miq-bot add_label gaprindashvili/yes

**Comment by juliancheal** [comment]
@miq-bot assign @bronaghs

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit 0d35c0addfb41d0c22a7a2d4c6963c6885830643
Author: Bronagh Sorota <bsorota@redhat.com>
Date:   Tue Jan 23 16:19:29 2018 -0500

    Merge pull request #402 from juliancheal/deletion_events
    
    Moved AWS_API_CALL_TerminateInstances from addition to deletion category
    (cherry picked from commit 1dc1215536b726ece28c8f04db06932f58321b7a)
    
    Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1537790
```

**Comment by juliancheal** [comment]
@miq-bot add_label fine/yes

**Pull request merged**